e2e-test/src/test/resources/pluginParameters.properties

```diff
@@ -0,0 +1,63 @@
+projectId=cdf-athena
+dataset=test_automation
+csvFormat=csv
+## FILE-PLUGIN-PROPERTIES-START
+incorrectFilePath=invalidPath
+csvFile=testdata/file/CSV_TEST.csv
+csvFileSchema=[{"key":"EmployeeDepartment","value":"string"},{"key":"Employeename","value":"string"},\
+  {"key":"Salary","value":"int"},{"key":"workhours","value":"int"}]
+csvFileOutputSchema={ "type": "record", "name": "text", "fields": [ \
+  { "name": "EmployeeDepartment", "type": "string" }, { "name": "Employeename", "type": "string" }, \
+  { "name": "Salary", "type": "int" }, { "name": "workhours", "type": "int" } ] }
+csvAllDataTypeFile=testdata/file/CSV_DATATYPE_TEST_1.csv
+csvAllDataTypeFileSchema=[{"key":"id","value":"int"},{"key":"name","value":"string"},\
+  {"key":"yearofbirth","value":"int"},{"key":"isdeleted","value":"boolean"},{"key":"email","value":"string"},\
+  {"key":"createddate","value":"string"},{"key":"revenue","value":"string"},{"key":"points","value":"string"},\
+  {"key":"longdatatype","value":"string"},{"key":"doubledatatype","value":"double"},\
+  {"key":"date","value":"string"},{"key":"null","value":"string"},{"key":"BytesData","value":"string"}]
+tsvFile=testdata/file/TSV_TEST.tsv
+tsvFileSchema=[{"key":"testscenarioid","value":"string"},{"key":"testdescription","value":"string"},\
+  {"key":"testconditionid","value":"string"},{"key":"testtype","value":"string"}]
+blobFile=testdata/file/BLOB_TEST.zip
+blobFileSchema=[{"key":"body","value":"bytes"}]
+delimitedFile=testdata/file/DELIMITED_TEST.txt
+delimiter=@
+incorrectDelimiter=#
+delimitedFileSchema=[{"key":"id","value":"int"},{"key":"name","value":"string"},\
+  {"key":"yearofbirth","value":"int"},{"key":"datatype","value":"long"},{"key":"tablename","value":"string"}]
+textFile=testdata/file/TEXT_TEST.txt
+textFileSchema=[{"key":"offset","value":"long"},{"key":"body","value":"string"}]
+outputFieldTestFile=testdata/file/OUTPUTFIELD_TEST.csv
+outputPathField=outputfield
+invalidOutputPathField=invalid
+outputFieldTestFileSchema=[{"key":"id","value":"int"},{"key":"firstname","value":"string"},\
+  {"key":"lastname","value":"string"},{"key":"photo","value":"string"},{"key":"outputfield","value":"string"}]
+overrideSchema=[{"key":"id","value":"float"},{"key":"firstname","value":"string"},\
+  {"key":"lastname","value":"string"},{"key":"photo","value":"string"},{"key":"outputfield","value":"string"}]
+overrideField=id
+invalidOverrideField=invalid
+overrideDataType=float
+readRecursivePath=testdata/file/RECURSIVE_TEST
+readRecursivePathSchema=[{"key":"id","value":"int"},{"key":"name","value":"string"},\
+  {"key":"yearofbirth","value":"int"},{"key":"tablename","value":"string"},{"key":"offset","value":"string"}]
+fileMaxSplitSize=120
+regexPathFilter=.+TEST.csv
+incorrectRegexPathFilter=/tZY
+filePluginOutputFolder=file-plugin-output
+skipHeaderTrue=true
+fileSinkPathSuffix=yyyy-MM-dd-HH-mm-ss
+## FILE-PLUGIN-PROPERTIES-END
+## DateTransform-PLUGIN-PROPERTIES-START
```

Comment:
Add separator empty lines--- removed in latest commit

Also add empty line after line no 3

Reply:
> Add separator empty lines--- removed in latest commit
> 
> Also add empty line after line no 3

added empty line